lib/ui/screens/groups_screen.dart

```diff
@@ -0,0 +1,183 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:reaxit/blocs/detail_state.dart';
+import 'package:reaxit/blocs/boards_cubit.dart';
+import 'package:reaxit/blocs/committees_cubit.dart';
+import 'package:reaxit/blocs/societies_cubit.dart';
+import 'package:reaxit/models/group.dart';
+import 'package:reaxit/ui/widgets/app_bar.dart';
+import 'package:reaxit/ui/widgets/error_scroll_view.dart';
+import 'package:reaxit/ui/widgets/group_tile.dart';
+import 'package:reaxit/ui/widgets/menu_drawer.dart';
+import 'package:collection/collection.dart';
+
+class GroupsScreen extends StatefulWidget {
+  final MemberGroupType? startScreen;
+
+  const GroupsScreen({this.startScreen});
+
+  @override
+  State<StatefulWidget> createState() => _GroupsScreenState();
+}
+
+class _GroupsScreenState extends State<GroupsScreen>
+    with SingleTickerProviderStateMixin {
+  late TabController _tabController;
+
+  @override
+  void initState() {
+    _tabController = TabController(length: 3, vsync: this);
```

Comment:
I think you can directly set the initial tab in the constructor, without animateTo

Reply:
https://gorouter.dev/nested-navigation here's some documentation about tabs in go_router btw